File: disambiguation/file_size/testSize.ts
function main() {
  const file = Deno.statSync("testSize.json");
  const fileSizeInBytes = file.size;
  console.log(`Deno, file size: ${fileSizeInBytes}`);

  const fileContent = Deno.readTextFileSync("testSize.json");
  const encoder = new TextEncoder();
  const bytes = encoder.encode(fileContent);
  console.log(`Deno, string size: ${bytes.byteLength}`);
}

main();

File: disambiguation/file_size/testSize.dart
import 'dart:convert';
import 'dart:io';

void main() {
  final file = File("testSize.json");
  final fileSizeInBytes = file.lengthSync();
  print("Dart: file size:  ${fileSizeInBytes}");

  final fileContent = file.readAsStringSync();
  final bytes = utf8.encode(fileContent);
  print("Dart: string size: ${bytes.length}");
}
